src/index.html

```diff
@@ -1,13 +1,44 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <main class="card" data-qa="card">
+      <div>
+        <img class="card_img" src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+        <div class="card_description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+        <div class="card_code">Product code: 195434</div>
```

Comment:
Wrong BEM naming
card is a block, code is a element 
so it supposed to be in __
```suggestion
        <div class="card__code">Product code: 195434</div>
```
pleas check this article 

https://en.bem.info/methodology/quick-start/

Reply:
Треба виправити лише в даному рядку ?

Reply:
У всьому коді, вірно ?